lib/datasource/pypi/index.ts

```diff
@@ -83,15 +83,39 @@ async function getDependency(
       }
     }
 
-    if (dep.info?.project_urls && !dependency.sourceUrl) {
+    if (dep.info?.project_urls) {
+      if (!dependency.sourceUrl) {
+        for (const [name, projectUrl] of Object.entries(
```

Comment:
Can you unify the loops, so we don't iterate twice

Reply:
maybe after #6420 is merged? there would be some merge conflics

Reply:
merged